src/Glpi/Controller/Config/Helpdesk/DeleteTileController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2025 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Controller\Config\Helpdesk;
+
+use CommonDBTM;
+use Config;
+use Glpi\Controller\AbstractController;
+use Glpi\Exception\Http\AccessDeniedHttpException;
+use Glpi\Exception\Http\BadRequestHttpException;
+use Glpi\Exception\Http\NotFoundHttpException;
+use Glpi\Helpdesk\HomePageTabs;
+use Glpi\Helpdesk\Tile\TileInterface;
+use Glpi\Helpdesk\Tile\TilesManager;
+use Glpi\Http\Firewall;
+use Glpi\Security\Attribute\SecurityStrategy;
+use Glpi\Session\SessionInfo;
+use Profile;
+use Session;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+use User;
```

Comment:
Some of these imports are probably not necessary.

Reply:
Somewhat unrelated but can we add unused import detection to the CI ?
I keep forgetting about them and finding files with lot of unused imports :/

I'm sure it was working in the CI at some point, why was it removed ?

Reply:
We had a specific packages for extra rules. Now we just use the PSR rules to be less strict.

Reply:
In another PR then.